understanding/understanding-techniques.html

```diff
@@ -100,40 +100,40 @@ <h2>Failures</h2>
 		</section>
 		<section>
 			<h2>Other Techniques</h2>
-			<p>In addition to the techniques in W3C's <a href="https://www.w3.org/WAI/WCAG21/Techniques/">Techniques for WCAG 2.1</a> document, <em>there are other ways to meet WCAG success criteria</em>. W3C's techniques are not comprehensive and may not cover newer technologies and situations.</p>
-			<p><em>Web content does not have to use W3C's published techniques in order to conform to WCAG 2.1.</em><em>(See also <a href="understanding-techniques-informative">Techniques are Informative</a> above.)</em></p>
-			<p>Content authors can develop different techniques. For example, an author could develop a technique for HTML5, <a href="https://www.w3.org/WAI/intro/aria">WAI-ARIA</a>, or other new technology. Other organizations may develop sets of techniques to meet WCAG 2.1 success criteria.</p>
+			<p>In addition to the techniques in W3C's <a href="../Techniques/">Techniques for WCAG 2</a> document, <em>there are other ways to meet WCAG 2 success criteria</em>. W3C's techniques are not comprehensive and may not cover newer technologies and situations.</p>
+			<p><em>Web content does not have to use W3C's published techniques in order to conform to WCAG 2. (See also <a href="understanding-techniques-informative">Techniques are Informative</a> above.)</em></p>
+			<p>Content authors can develop different techniques. For example, an author could develop a technique for HTML5, <a href="https://www.w3.org/WAI/intro/aria">WAI-ARIA</a>, or other new technology. Other organizations may develop sets of techniques to meet WCAG 2 success criteria.</p>
 			<p>Any techniques can be sufficient if:</p>
 			<ul>
 				<li>
 					<p>they satisfy the success criterion, and</p>
 				</li>
 				<li>
-					<p> all of the <a href="conformance">WCAG 2.1 conformance requirements</a> are met.</p>
+					<p> all of the <a href="conformance">WCAG 2 conformance requirements</a> are met.</p>
 				</li>
 			</ul>
 			<section>
 				<h3>Submitting Techniques</h3>
-				<p>The WCAG Working Group encourages people to submit new techniques so that they can be considered for inclusion in updates of the <a href="https://www.w3.org/WAI/WCAG21/Techniques/">Techniques for WCAG 2.1</a> document. Please submit techniques for consideration using the <a href="https://www.w3.org/WAI/GL/WCAG20/TECHS-SUBMIT/">Techniques Submission Form</a>.</p>
+				<p>The WCAG Working Group encourages people to submit new techniques so that they can be considered for inclusion in updates of the <a href="../Techniques/">Techniques for WCAG 2</a> document. Please submit techniques for consideration to the <a href="https://github.com/w3c/wcag/">WCAG source repository</a>.</p>
```

Comment:
```suggestion
				<p>The WCAG Working Group encourages people to submit new techniques so that they can be considered for inclusion in updates of the <a href="../Techniques/">Techniques for WCAG 2</a> document. Please submit techniques for consideration to the <a href="https://github.com/w3c/wcag/">WCAG repository on GitHub</a>.</p>
```
Changing 'WCAG source repository' to 'WCAG repository on GitHub'. CC @patrickhlauke